jetty-ee9/jetty-ee9-proxy/src/main/java/org/eclipse/jetty/ee9/proxy/AbstractProxyServlet.java

```diff
@@ -768,10 +771,9 @@ protected void sendProxyResponseError(HttpServletRequest clientRequest, HttpServ
         }
     }
 
-    protected void onContinue(HttpServletRequest clientRequest, Request proxyRequest)
+    protected Runnable onContinue(HttpServletRequest clientRequest, Request proxyRequest)
```

Comment:
A little javadoc explaining the return would be good, specifically that null makes it optional